Chinese/DesertHell/OldStory/Story.md

```diff
@@ -5,4 +5,4 @@
 很久很久以前,
 有一隻白猫,
 他是一隻活过一万次的猫,
-[他讲的什么？](./Story.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.